### PR TITLE
Sqlite integration

### PR DESCRIPTION
**Pull request by ghost**
SQLite integration done by npm package sqlite3, now site has a admin model named superUser and a trivial model active_member. 
only the sessions with name "superuser" can access "/admin" page and things that can be accessible from there.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,10 +31,80 @@ app.get('/*', function (req, res, next) {
...
+  var username = req.body.username;
+  var password = req.body.password;
+
+  db.get("SELECT EXISTS(SELECT * FROM superusers WHERE username='"+ username +"')",function(err,result){
+    console.log(result);
+    console.log(result[Object.keys(result)[0]])
+    if( result[Object.keys(result)[0]] == 1 ){
+      db.get("SELECT * FROM superusers WHERE username='"+ username +"'", (err, res1) => {
+        console.log(res1)
+        if( password == res1.password ){
```
Comment (duck2) [inline index.js]: hehehe. maybe https://www.owasp.org/index.php/Password_Storage_Cheat_Sheet?

File: index.js [hunk 2]
```diff
@@ -55,24 +54,25 @@ app.get('/admin/login',function(req,res){
...
-          req.session.name = "superuser"
-          res.redirect('/admin')
-        }else {
-          res.redirect('/')
-        }
-      })
-    }else {
-      res.redirect('/')
-    }
+  var hashed = crypto.createHmac('sha256',req.body.password).digest('hex');
```
Comment (duck2) [inline index.js]: Crypto has a _key derivation_ API for hard-hashing stuff(~100000 times), it's better in this case.

https://nodejs.org/api/crypto.html#crypto_crypto_pbkdf2_password_salt_iterations_keylen_digest_callback

**Comment by duck2** [comment]
couldn't see anything else wrong, i'll try to run this in local after magnet exam today

**Comment by ghost** [comment]
Looks fine, should we merge?

**Pull request merged**